easybuild/easyconfigs/s/Sphinx/Sphinx-2.4.3-GCCcore-8.3.0-Python-3.7.4.eb

```diff
@@ -0,0 +1,86 @@
+easyblock = 'PythonBundle'
+
+name = 'Sphinx'
+version = '2.4.3'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://sphinx.pocoo.org/'
+description = """Sphinx is a tool that makes it easy to create intelligent and beautiful documentation.
+ It was originally created for the new Python documentation, and it has excellent facilities for the 
+ documentation of Python projects, but C/C++ is already supported as well, and it is planned to add 
+ special support for other languages as well."""
+
+toolchain = {'name': 'GCCcore', 'version': '8.3.0'}
+
+builddependencies = [('binutils', '2.32')]
+
+dependencies = [('Python', '3.7.4')]
+
+use_pip = True
+
+exts_list = [
+    ('packaging', '20.1', {
+        'source_urls': ['https://pypi.python.org/packages/source/p/packaging/'],
```

Comment:
@furstj You can avoid including `source_urls` for every extension using `exts_default_options` above `exts_list`:

```python
exts_default_options = {'source_urls': [PYPI_SOURCE]}

exts_list = [
    ('packaging', '20.1', {
        'checksums': ['e665345f9eef0c621aa0bf2f8d78cf6d21904eef16a93f020240b704a57f1334'],
    }),
    ('MarkupSafe', '1.1.1', {
        'checksums': ['29872e92839765e546828bb7754a68c418d927cd064fd4708fab9fe9c8bb116b'],
    }),
    ...